.github/workflows/schema-bq.yml

```diff
@@ -0,0 +1,54 @@
+# This workflow is used to push changed HTAN.model.csv 
+# to BigQuery table located at `htan-dcc.metadata.data-model`
+
+name: Add updated schema to BigQuery
+
+on:
```

Comment:
can we add `workflow_dispatch:` to the `on` action trigger list. This enables us to manually trigger the action when needed.